pkg/addon/configpolicy/manifests/managedclusterchart/templates/policy.open-cluster-management.io_operatorpolicies_crd.yaml

```diff
@@ -354,6 +354,14 @@ spec:
                 resolvedSubscriptionLabel:
                   description: The resolved name.namespace of the subscription
                   type: string
+                subscriptionInterventionTime:
+                  description: |-
+                    Timestamp for a possible intervention to help a Subscription stuck with a
+                    ConstraintsNotSatisfiable condition. Can be in the future, indicating the
```

Comment:
Um yeah, this isn't a complete sentence. 😉 

>Timestamp for a possible intervention to help a Subscription stuck with a ConstraintsNotSatisfiable condition

Reply:
You're not wrong. I'll try to draft something better